.env

```diff
@@ -0,0 +1,4 @@
+COMPOSE_PROJECT_NAME=facets-knowledge-panels
+API_EXPOSE=127.0.0.1:80
+COMPOSE_PATH_SEPARATOR=; 
+COMPOSE_FILES=docker-compose.yml;docker/dev.yml
```

Comment:
We usually make "sections" al that is about docker compose comes first. App configuration after that.

```suggestion
COMPOSE_PATH_SEPARATOR=; 
COMPOSE_FILES=docker-compose.yml;docker/dev.yml

# port expose for the API
API_EXPOSE=127.0.0.1:80
```